packages/gestalt/src/TableOfContents/TableOfContentsAnchor.css

```diff
@@ -0,0 +1,16 @@
+.item {
+  border-radius: 2px 8px 8px 2px;
+  padding-right: 16px;
```

Comment:
None of these seem to be using our spacing tokens. Anyway we can use css composes for spacing and use our padding tokens so we can have this work with density theme in the future